cogs5e/sheets/errors.py

```diff
@@ -6,7 +6,35 @@
 
 
 class MissingAttribute(Exception):
```

Comment:
We should have these error classes inherit from `ExternalImportError` (which iirc didn't exist when I originally wrote this? 😛 )